dataland-backend/build.gradle.kts

```diff
@@ -75,7 +75,7 @@ tasks.test {
     useJUnitPlatform()
 
     extensions.configure(JacocoTaskExtension::class) {
-        setDestinationFile(file("$buildDir/jacoco/jacoco.exec"))
```

Comment:
This change i don't understand. On my local machine it works without these changes